homeworks/Aleksandr.Turuta_turuta7/homework_7/homework.test.js

```diff
@@ -0,0 +1,19 @@
+const { cycleAndTest } = require('./homework_7');
+
+describe('test fibonacci', () => {
```

Comment:
most of time `describe()` method first argument should be the tested unit name, e.g. `describe('cycleAndTest()'...`